### PR TITLE
(grc_conversion) EpiDoc and CTS conversion work

### PR DESCRIPTION
**Pull request by lcerrato**
see #1399 
continuing with tlg0648.tlg001 file conversion

### CODE REVIEW

File: data/tlg0652/tlg001/tlg0652.tlg001.perseus-grc2.xml
[the remark this answers is not in the document]
Comment (lcerrato) [reply data/tlg0652/tlg001/tlg0652.tlg001.perseus-grc2.xml:1]: Sure. Good catch.

File: data/tlg0653/tlg001/tlg0653.tlg001.perseus-grc2.xml
Comment (AlisonBabeu) [inline data/tlg0653/tlg001/tlg0653.tlg001.perseus-grc2.xml:1]: Missing an xml:lang tag on `<author>Aratus Solensis</author>` in the structured bibliographic info.

**Review by AlisonBabeu — APPROVED** [review]
hi @lcerrato all of these files look good!

**Review by AlisonBabeu — APPROVED** [review]

**Comment by AlisonBabeu** [comment]
hi @lcerrato I checked out the rest of the files and the dates and don't see any errors!

**Pull request merged**